lib/api/controllers/documentController.js

```diff
@@ -173,6 +174,45 @@ class DocumentController extends NativeController {
     };
   }
 
+  async export (request) {
+    const { index, collection } = request.getIndexAndCollection();
+    const { size, scrollTTL, searchBody } = request.getSearchParams();
+    const format = request.getString('format', 'jsonl');
+    const fields = request.getBodyArray('fields', []);
+    const lang = request.getLangParam();
+    const separator = request.getString('separator', ',');
+
+    if (request.context.connection.protocol !== 'http') {
+      throw kerror.get('api', 'assert', 'unsupported_protocol', request.context.connection.protocol, 'document:export');
+    }
+
+    if (lang === 'koncorde') {
+      searchBody.query = await this.translateKoncorde(searchBody.query || {});
+    }
+
+    let mimeType = 'html/text';
+
+    if (format.toLowerCase() === 'csv') {
+      mimeType = 'text/csv';
+    }
+
+    request.response.setHeader('Content-Type', mimeType);
+    request.response.setHeader('Content-Disposition', `attachment; filename="${index}-${collection}.${format}"`);
+
+    return dumpCollectionDocument(
+      index,
+      collection,
+      searchBody,
+      format,
+      fields,
+      {
+        lang,
+        scroll: scrollTTL || '1m',
```

Comment:
Since the default size is `10` documents per page, I think we can safely have a smaller value like `5s` for the scroll